packages/oruga/src/composables/useInputHandler.ts

```diff
@@ -61,10 +73,9 @@ export function useInputHandler(
     // inject parent field component if used inside one
     const { parentField } = injectField();
 
-    const element = computed<ValidatableFormElement>(() => {
+    const maybeElement = computed<ValidatableFormElement | undefined>(() => {
         const el = unrefElement<Component | HTMLElement>(inputRef);
         if (!el) {
-            console.warn("useInputHandler: inputRef contains no element");
```

Comment:
Because you run the watcher on immediate `{ immediate: true },`. Maybe this isn't necessary? While not having immediate, isn't the watcher called after mount, when the element ref is filled?

Otherwise, please add a comment to `maybeElement` about why it exists.

Reply:
> While not having immediate, isn't the watcher called after mount, when the element ref is filled?

That's the most likely trigger for the watcher, but since it watches multiple values, it may still be triggered before the component is mounted. In particular, changes to `props.useHtml5Validation` might theoretically trigger the watcher prematurely.

By calling the [`onMounted` hook](https://vuejs.org/api/composition-api-lifecycle.html#onmounted), I theoretically could defer the creation of the watcher until the element ref has been populated, but if the component is inside a `<KeepAlive>`, the element ref may later become empty because the component [is detached from the DOM](https://vuejs.org/guide/built-ins/keep-alive.html#lifecycle-of-cached-instance).

I've [added a comment](https://github.com/oruga-ui/oruga/pull/930/commits/e17a4fa0581c561a72441ec4ddf3178a4c8036ea) detailing when we expect the ref to be empty.